lenovo/thinkpad/x1/6th-gen/default.nix

```diff
@@ -2,7 +2,9 @@
 # standing with kernel 4.6.11 is the following wiki page:
 # https://wiki.archlinux.org/index.php/Lenovo_ThinkPad_X1_Carbon_(Gen_6). The
 # TrackPoint and TouchPad issues there seem to have been fixed already.
-
+#
+# Enable the lower-power S3 suspend state by upgrading UEFI to version >= 1.30,
```

Comment:
Maybe it's better to use BIOS instead of UEFI here?

Reply:
Yeah that's a good point; upgraded comment accordingly.